SPECS-EXTENDED/libproxy/libproxy.signatures.json

```diff
@@ -1,6 +1,5 @@
 {
  "Signatures": {
-  "libproxy-0.4.17.tar.gz": "88c624711412665515e2800a7e564aabb5b3ee781b9820eca9168035b0de60a9",
-  "proxy.1": "e53e8d6872d707b8a50994976c7e2502d3a104d677db57210592afd459047497"
+  "0.5.8.tar.gz": "64e363855012175bf796b37cacddf7bc7e08af0bf406eea94b549ce207987d3e"
```

Comment:
Keep the source naming convention to add distinction of package name. version string is not unique.